### DISCUSSION TITLE
Welcome to chatgpt-ui Discussions!

### DISCUSSION THREAD

**Discussion by WongSaang** [WongSaang/chatgpt-ui#24] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cgofun** [comment]
The response speed of the chat is very slow and the real-time performance is poor. It takes several tens of seconds for the AI to reply. The API request speed test is not slow. What could be the reason for this?

**Reply by spencerwongfeilong** [reply]
I agree with your observation. I tried other chatgpt instances in github and they were faster. 

@WongSaang let us know what kind of tests we can do to help arrest the issue. Thank you!

**Reply by WongSaang** [reply]
Hello,

I have done some work to solve this issue. Please update both the front-end and back-end images to the latest version and try again.
